test/DynamoCoreTests/CallsiteTests.cs

```diff
@@ -162,5 +164,49 @@ public void Callsite_ElementBinding()
             AssertPreviewValue("c760af7e-042c-4722-a834-3445bf41f549", 2);
             AssertPreviewValue("5f277520-13aa-4833-aa82-b17a822e6d8c", 3);
         }
+
+        [Test]
+        public void Callsite_ElementBinding_Timing()
+        {
+            //This graph loads trace data for 1500 "WrapperObjects" in Manual run mode.
+            var ws = Open<HomeWorkspaceModel>(TestDirectory, callsiteDir, "element_binding_large.dyn");
+            var sw = new Stopwatch();
+            sw.Start();
+
+            BeginRun();
+            sw.Stop();
+            Assert.Less(sw.Elapsed.Milliseconds, 20000);
```

Comment:
How much time would this take earlier?

Reply:
I will check - with the same files I was using with similar sized data it took 2.5 minutes.

Reply:
Sorry, it took over a min when I ran it again. I might have made a mistake running it the first time. It looked a lot faster then. I think testing for <20s shouldn't be a problem on most machines if you claim that it actually takes just 2s.

Reply:
The tests also do some really simple element binding and element disposal testing.